appveyor.yml

```diff
@@ -14,16 +14,19 @@ init:
 
 before_build:
   - ps: .\build\Set-BuildVersion -Version $env:APPVEYOR_BUILD_VERSION -Suffix $env:VERSION_SUFFIX
-  - appveyor-retry dotnet restore -v Minimal
+  - appveyor-retry dotnet restore -v Minimal  
 
 build_script:
-  - ps: dotnet build -c Release
+  - ps: dotnet build -c Release 
 
+test_script:

```

Comment:
Yeah, I did run into an issue when I was trying to run them with dotnet test.  It worked inside of Visual Studio, but I wanted to make sure it would work if appveyor was using dotnet test internally.  The error I got was "Make sure test project has a nuget reference of package "microsoft.testplatform.testhost".  I'll go ahead and revert it to the original version and see how that goes.